### DISCUSSION TITLE
Make area start at the bottom of the graph instead of 0 on Area chart?

### DISCUSSION THREAD

**Discussion by bjhorrell** [jfree/jfreechart#383] [Q&A]
I would like to make my Area Chart start at the lowest Y value instead of 0 like this.
Is this possible, Thanks in advance.
![after](https://github.com/jfree/jfreechart/assets/93740148/e4d383ed-48e6-4e74-99d7-a0c92f3d7643)

**Comment by trashgod** [comment]
You can set the lower bound explicitly, _e.g._  `plot.getRangeAxis().setLowerBound(0)`.

**Reply by bjhorrell** [reply]
Sorry, I meant the areas starting at the bottom of the graph, instead of revolving around 0 like they do in this picture.
![before](https://github.com/jfree/jfreechart/assets/93740148/ac1e0426-0c17-4e31-b050-5ea3e5dfb776)

**Comment by trashgod** [comment]
Can you use one of the `DatasetUtils` to  find the desired bound?

**Reply by bjhorrell** [reply]
I know what the lower bound of my graph is(in this case it's is -10). I just can figure out how to get my areas to start from there instead of 0.

**Reply by trashgod** [reply]
AFAIK, the renderer can only display values in the dataset. Does a third dataset make any sense for this use-case? Cross-posted [here](https://stackoverflow.com/q/77390459/230513).
